test/TBD/specialization.swift

```diff
@@ -1,5 +1,5 @@
 // REQUIRES: VENDOR=apple 
-// Validate the the specializations actually exist (if they don't then we're not
+// Validate the specializations actually exist (if they don't then we're not
```

Comment:
Still a fragment. Perhaps
```suggestion
// Validate that the specializations actually exist (if they don't then we're not
```